pom.xml

```diff
@@ -61,5 +66,6 @@
     <module>org.eclipse.jdt.launching.ui.macosx</module>
     <module>org.eclipse.jdt.debug.tests</module>
     <module>org.eclipse.jdt.debug.ui</module>
+    <module>org.eclipse.jdt.debug.jdi.tests</module>
```

Comment:
Since org.eclipse.jdt.debug.jdi.tests doesn't contain a pom.xml and not a maven project, does this work? We may need to convert it to a maven project first.

Reply:
Yes, will have to add that. Let me add in this PR and then you include in your PR as it may not succeed without the changes.

Reply:
You can try with the added pom file from this PR.

Reply:
Do I need to wait for your PR to merge first and then rebase my PR? Or I just copy the same changes to it?

Reply:
You include these 2 changes to your PR.
I am not planning to release this PR.

Reply:
got it. thanks.